protobuf/schema/v1/schema.proto

```diff
@@ -10,6 +10,39 @@ option php_namespace = "OpenFeature\\Providers\\Flagd\\Schema\\Grpc";
 import "google/protobuf/struct.proto";
 import "google/protobuf/empty.proto";
 
+// Request body for bulk flag evaluation, used by the ResolveAll rpc.
+message ResolveAllRequest {
+
+    // Object structure describing the EvaluationContext used in the flag evaluation, see https://docs.openfeature.dev/docs/reference/concepts/evaluation-context
+    google.protobuf.Struct context = 1;
+}
+
+// Response body for bulk flag evaluation, used by the ResolveAll rpc.
+message ResolveAllResponse {
+
+    // Object structure describing the evaluated flags for the provided context.
+    map<string, AnyFlag> flags = 1;
```

Comment:
Is `flags` good enough? Or should we call it `evaluatedFlags` or `evaluated`?

Reply:
Im happy with `flags`